### ISSUE TITLE
About the mAP

### ISSUE THREAD

**Issue by Yvvonne** [HiKapok/SSD.TensorFlow#16]
I trained the VOC dataset, but only got 64% mAP. Is there any trick do you use to reach the report map (77%) without the pertained model?

**Comment by HiKapok** [comment]
@Yvvonne No, did you use the latest codes? As far as I know, there were a lot peoples trained SSD using this codes successfully

**Comment by Yvvonne** [comment]
Yes, I already use the newest code. Maybe because I didn't use the pre-trained model? And I also added the warm-up codes as you suggest.

**Comment by HiKapok** [comment]
@Yvvonne I'm sorry for that I didn't read your issues carefully. It's easy to get 77%+mAP with the pre-trained weights of vgg16 provided in readme.md, without any tricks.
